app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,33 @@
+class MenusController < ApplicationController
+  before_action :set_menu, only: [:show, :edit, :update]
+  load_and_authorize_resource
```

Comment:
I believe this should be placed in application controller. then `skip_load_and_authorize_resource` when needed